examples/chart/teleport/templates/service.yaml

```diff
@@ -3,11 +3,11 @@ kind: Service
 metadata:
   name: {{ .Chart.Name }}
 spec:
-  type: LoadBalancer
+  type: ClusterIP
```

Comment:
it would be helpful to make this configurable, because I usually need load balancer when setting up on GKE, I imagine in many cases that's not necessary, same about ports?

Reply:
Makes sense! Actually I need to run Teleport on K8S on AWS so I'll make necessary changes for that, which should also address your use-case.